standard/namespaces.md

```diff
@@ -446,10 +449,130 @@ Because names may be ambiguous when more than one imported namespace introduces
 > ```
 > *end example*
 
+Furthermore, when more than one namespace or type imported by *using_namespace_directive*s or *using_static_directive*s in the same compilation unit or namespace body contain types or members by the same name, references to that name as a *simple_name* are considered ambiguous.
+
+> *Example*:
+> ```csharp
+> namespace N1
+> {
+>     class A {}
+> }
+> class C
+> {
+>     public static int A;
+> }
+> namespace N2
+> {
+>     using N1;
+>     using static C;
+>     class B
+>     {
+>         void M() 
+>         { 
+>             A a = new A();   // Ok, A is unambiguous as a type-name
+>             A.Equals(2);     // Error, A is ambiguous as a simple-name
+>         }
+>     }
+> }
+> ```
+> `N1` contains a type member `A`, and `C` contains a static field `A`, and because `N2` imports both, referencing `A` as a *simple_name* is ambiguous and a compile-time error. *end example*
+
 Like a *using_alias_directive*, a *using_namespace_directive* does not contribute any new members to the underlying declaration space of the compilation unit or namespace, but, rather, affects only the compilation unit or namespace body in which it appears.
 
 The *namespace_name* referenced by a *using_namespace_directive* is resolved in the same way as the *namespace_or_type_name* referenced by a *using_alias_directive*. Thus, *using_namespace_directive*s in the same compilation unit or namespace body do not affect each other and can be written in any order.
 
+### §using-static-directives Using static directives
+
+A *using_static_directive* imports the nested types and static members contained directly in a type declaration into the immediately enclosing compilation unit or namespace body, enabling the identifier of each member and type to be used without qualification.
+
+```antlr
+using_static_directive
+    : 'using' 'static' type_name ';'
+    ;
+```
+Within member declarations in a compilation unit or namespace body that contains a *using_static_directive*, the accessible nested types and static members (except extension methods) contained directly in the declaration of the given type can be referenced directly.
+
+> *Example*:
+> ```csharp
+> namespace N1
+> {
+>    class A 
+>    {
+>         public class B{}
+>         public static B M(){ return new B(); }
```

Comment:
Can we use an expression-bodied method for this? If not, can we have a space between `()` and `{`?